.github/PULL_REQUEST_TEMPLATE/release-pull-request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Release
+about: Provide specifics about a PR for a release
+title: 'PR/Release: ____(provide a descriptive name)'
+assignees: ''
+
+---
+<!-- IMPORTANT: Please do not create a PR without creating an issue first. -->
+
+<!-- Fields in **bold** are REQUIRED, fields in *italics* are OPTIONAL. -->
+
+**Description**
+<!-- A description of the updates contained in this release. Example:  -->
+<!-- Incorporates Travis CI feature #91, summary for script_run #99, and root portal handling #88 -->
+
+Closes #
+
+*Screenshots or additional context*
+<!-- Add any other context about this release. -->
+
+**Bump version information**
+Bump version: v0.xx.x -> v0.xx.0
+
+Bumped in:
+- [ ] README.md, [installation instructions](../../README.md#installation-instructions)
+- [ ] README.md, [changelog](../../README.md#changelog)
```

Comment:
This isn't really a bump but a changelog update. Should we put it separate it from Bump version listing?

Reply:
I added a Chaneglog section to all the PR templates with instructions specific for each type of PR